source/deploy/mobile-faq.rst

```diff
@@ -372,3 +372,16 @@ In these scenarios, the ``login`` audit log won't have the ``deviceId``, and the
 If the app receives the ``deviceId`` later, the device will send the new ``deviceId`` to the server, generating an ``attachDeviceId`` audit log, and adding the ``deviceId`` to the session data in the database.
 
 Since the ``deviceId`` relates to the application, connections through the web browser, even on mobile, won't have a ``deviceID``.
+
+Where can I find mobile message notification logs?
+-------------------------------------------------------------
+Notification messages are logged to the ``notifications.log`` file.
+System admins can only access this ``notifications.log`` file by going to the ``File Location`` set in the ``Notification Settings``. The admins also have the privilege to set this file location and in case, no file location has been set, then the ``notifications.log`` file gets stored in the default working directory.
+Also, the ``notifications.log`` file exists only when it is enabled in Mattermost by setting ``NotificationSettings.EnableFile`` to ``true`` in the ``config.json`` file.
```

Comment:
This text has been incorporated into the proposed text directly above and is no longer needed. Please remove.